windows/CodePushLifecycleEventListener.cs

```diff
@@ -0,0 +1,44 @@
+﻿using ReactNative.Bridge;
+using System;
+
+namespace CodePush.ReactNative
+{
+    internal class CodePushLifecycleEventListener : ILifecycleEventListener
+    {
+        private DateTime? _lastSuspendDate = null;
+        private Action _loadBundleAction;
+
```

Comment:
I would keep the name of this agnostic to what it actually does, that way it's fully decoupled from the consumer. Maybe just "_resumeAction"?